### PR TITLE
Add mega drop shadow

### LINKED ISSUE #222: [BUG] mega menu drop shadow required
## Description
When opening the mega menu on dev, the lack of bottom drop shadow makes it hard to delineate between other blue sections. The drop shadow adds this needed separation

## Screenshots
screen shot 2018-07-22 at 00 56 52

### PR DESCRIPTION
**Pull request by mtuchi**
## Description

Add box-shadow in a mega dropdown, Quick note: The changes are significantly small to see

Fixes #222 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
shadow

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/templates/takwimu/_includes/navbar/_dropdown.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 <div class="dropdown-menu-lg text-light py-5 container-fluid {% block dropdown_class %}{% endblock %}"
     id="{% block dropdown_id %}{% endblock %}"
-    style="{% block dropdown_style %}{% endblock %}">
+    style="{% block dropdown_style %}{% endblock %} box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.24);">
```
Comment (DavidLemayian) [inline takwimu/templates/takwimu/_includes/navbar/_dropdown.html]: The `shadow` class doesn't do this for us?

**Comment by DavidLemayian** [comment]
@mtuchi Also keep assignment to one person (@kilemensi) to avoid confusion on who's responsible to have it merged.

File: takwimu/templates/takwimu/_includes/navbar/_dropdown.html (hunk 1 above)
Comment (mtuchi) [reply takwimu/templates/takwimu/_includes/navbar/_dropdown.html]: @DavidLemayian the `shadow` class give us `box-shadow: 0 .5rem 1rem rgba(0,0,0,.15) !important;`
The zeplin design is `box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.24)`
Should use the `shadow` class instead ?

Comment (DavidLemayian) [reply takwimu/templates/takwimu/_includes/navbar/_dropdown.html]: @mtuchi Yes, let's use shadow class instead for standard look and feel with Bootstrap right now.

**Pull request merged**